modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -53,36 +53,43 @@ def parse_table_row(row)
     end
 
     def parse_form_row(line, url)
-      title = line.css('font').text
       form_name = line.css('a').first.text
       form = VaForms::Form.find_or_initialize_by form_name: form_name
       current_sha256 = form.sha256
-      form.title = title
-      issued_string = line.css('td:nth-child(3)').text
+      form.title = line.css('font').text
       revision_string = line.css('td:nth-child(4)').text
-      form.first_issued_on = parse_date(issued_string) if issued_string.present?
       form.last_revision_on = parse_date(line.css('td:nth-child(4)').text) if revision_string.present?
       form.pages = line.css('td:nth-child(5)').text
-      form_url = url.starts_with?('http') ? url : get_full_url(url)
+      form_url = url.starts_with?('http') ? url.gsub('http:', 'https:') : get_full_url(url)
```

Comment:
how about `sub` over `gsub`? should url be downcased first?

Reply:
I don't want to mess with the casing because not all VA systems are clean enough to understand downcased vs upcased etc. Leaving it as it is makes no assumptions as to the nature of the applications hosting these pdfs and gsub ensures that the linear structure of the swap is maintained moving forward.